apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -340,6 +343,19 @@ public PersistenceAnnotationBeanPostProcessor persistenceAnnotationProcessor() {
     return stu3ResourceProviders;
   }
 
+  /**
+   * Determines if the fhir resources related to pre adj claim data should be accessible via the
+   * fhir api service.
+   *
+   * @return True if the resources should be available to consume, False otherwise.
+   */
+  private static boolean isPreAdjResourcesEnabled() {
+    return Boolean.TRUE
+        .toString()
+        .equalsIgnoreCase(
+            ObjectUtils.defaultIfNull(System.getProperty("bfdServer.preadj.enabled"), "false"));
```

Comment:
Meaning would be clearer using `System.getProperty("bfdServer.preadj.enabled", "false")`